examples/pygstc/simple_pipeline.py

```diff
@@ -0,0 +1,83 @@
+#The software contained in this file is free and unencumbered software released into the public domain.
+#Anyone is free to use the software contained in this file as they choose, including incorporating
+#it into proprietary software.
+import time
+import sys
+from pygstc.gstc import *
+from pygstc.logger import *
+
+#Create a custom logger with loglevel=DEBUG
+gstd_logger = CustomLogger('simple_pipeline', loglevel='DEBUG')
+
+#Create the client with the logger
+gstc = GstdClient(logger=gstd_logger)
+
+def printError():
+    print("To play run: python3 simple_pipeline.py create $VIDEO_PATH")
+    print("To play run: python3 simple_pipeline.py play")
+    print("To pause run: python3 simple_pipeline.py pause")
+    print("To stop run: python3 simple_pipeline.py stop")
+    print("To reverse play run: python3 simple_pipeline.py reverse")
+    print("To play slow run: python3 simple_pipeline.py slow_motion")
+    print("To stop run: python3 simple_pipeline.py delete")
+    print("To read the bus run: python3 simple_pipeline.py read_bus")
```

Comment:
You are creating a high level application. Hide the concept of the bus and the EOS.